src/Components/Reports/SystemsPDFReport.js

```diff
@@ -1,51 +1,72 @@
+export const DownloadSystemsPDFReport = ({ filters, orderBy, orderHow, ...props }) => {
+    const reportFileName = getSystemsReportFileName();
+    const [addNotification, clearNotifications] = useNotification();
+    const { start, success, failure } = REPORT_NOTIFICATIONS;
 
-    const { systemsReportFirstPage, systemsReportRestPages } = PDF_REPORT_PER_PAGE;
+    useEffect(() => {
+        addNotification(start);
+    }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
-    // Table rows
-    const fetchSystemParams = {
-        filters,
-        stateFilter: filters.stateFilter,
-        orderBy,
-        orderHow
-    };
-    const systemsResponse = await fetchSystems(fetchSystemParams);
-    const pdfData = formatData(systemsResponse.data, 'pdf');
+    const generateSystemsPDFReport = async (filters, orderBy, orderHow) => {
 
-    // first page description and data
-    const totalSystems = systemsResponse?.meta?.count;
-    const filterText = generateFilterText(filters);
-    const firstPageData =  pdfData.splice(0, systemsReportFirstPage);
+        const { systemsReportFirstPage, systemsReportRestPages } = PDF_REPORT_PER_PAGE;
 
-    const firstPage = <SystemsFirstPage
-        data={firstPageData}
-        totalSystems={totalSystems}
-        filterText={filterText} />;
+        // Table rows
+        const fetchSystemParams = {
+            filters,
+            stateFilter: filters.stateFilter,
+            orderBy,
+            orderHow
+        };
 
-    const otherPages = [];
+        let systemsResponse;
+        try {
+            systemsResponse = await fetchSystems(fetchSystemParams);
+        }
+        catch {
+            clearNotifications();
+            addNotification(failure);
 
-    while (pdfData.length > 0) {
-        otherPages.push(pdfData.splice(0, systemsReportRestPages));
-    }
+            return [];
+        }
 
-    return [
-        firstPage,
-        ...otherPages.map((systemsPage, index) => <SystemsTablePage key={index} data={systemsPage}  page={index + 1}/>)
-    ];
+        const pdfData = formatData(systemsResponse.data, 'pdf');
 
-};
+        // first page description and data
+        const totalSystems = systemsResponse?.meta?.count;
+        const filterText = generateFilterText(filters);
+        const firstPageData =  pdfData.splice(0, systemsReportFirstPage);
 
-export const DownloadSystemsPDFReport = ({ filters, orderBy, orderHow, ...props }) => {
-    const reportFileName = getSystemsReportFileName();
+        const firstPage = <SystemsFirstPage
+            data={firstPageData}
+            totalSystems={totalSystems}
+            filterText={filterText} />;
+
+        const otherPages = [];
+
+        while (pdfData.length > 0) {
+            otherPages.push(pdfData.splice(0, systemsReportRestPages));
+        }
+
+        clearNotifications();
+        addNotification(success);
+
+        return [
+            firstPage,
```

Comment:
```suggestion
            <SystemsFirstPage {...firstPageProps} />,
```

Reply:
I will push the changes in the latest commit 👍🏻